HW03/anastasia_evsihneyeva/spec/anastasia_evsihneyeva_hw_03_t_01_spec.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+require_relative '/home/anastasia/Документы/ruby/homeworks-2022/spec/spec_helper'
+require_relative '/home/anastasia/Документы/ruby/homeworks-2022/HW03/anastasia_evsihneyeva/anastasia_evsihneyeva_hw_03_t_01'
+
+RSpec.describe Homework3 do
+  first_excersise_variable = described_class.new
```

Comment:
please avoid using local variables in spec.
 change this to something like: subject { described_class.configuration }